docs/source/02_get_started/04_new_project.md

```diff
@@ -1,6 +1,6 @@
 # Create a new project
 
-Once you have [installed Kedro](./02_install.md), you can create a new project by answering a series of questions, or by using settings recorded in a configuration file.
+Once you have [installed Kedro](./02_install.md), you can create a new project interactively by answering a series of questions, or by using settings recorded in a configuration file.
```

Comment:
```suggestion
Once you have [installed Kedro](./02_install.md), you can create a new project by answering a series of questions, or by using settings recorded in a configuration file.
```

My feeling is that answering a series of questions is, by its nature, interactive. Thus redundant to state it.

Reply:
This is just based on my own experience. When I read the heading 'Create a new project interactively', I had to re-read the previous line to see if I had missed something, and spent several seconds wondering if the series of questions was the same as the interaction.